src/plugins/plugin-mysql2.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var shimmer = require('shimmer');
+
+var SUPPORTED_VERSIONS = '^1.3.5';
+
+function createCreateQueryWrap(api) {
+  return function createQueryWrap(createQuery) {
+    return function createQuery_trace(sql, values, cb, config) {
```

Comment:
requires an additional `config` argument

Reply:
I believe this is unnecessary if `config` is not being used. We already make a call to `createQuery` using the `arguments` object.

Reply:
In which case non of those arguments are really necessary, correct?

Reply:
Ahh, yeah. For some reason I thought `cb` was being used which would be why I thought the args were spelled out in the first place.

I guess they aren't necessary but let's keep it for readability's sake.